test/integrations/sources/auth0/data.ts

```diff
@@ -9,240 +9,247 @@ export const data = [
-                        'update:email_templates',
-                        'update:email_triggers',
-                        'update:guardian_factors',
-                        'update:integrations',
-                        'update:log_streams',
-                        'update:mfa_policies',
-                        'update:organization_connections',
-                        'update:organizations',
-                        'update:prompts',
-                        'update:requested_scopes',
-                        'update:resource_servers',
-                        'update:roles',
-                        'update:rules',
-                        'update:rules_configs',
-                        'update:shields',
-                        'update:signing_keys',
-                        'update:tenant_members',
-                        'update:tenant_settings',
-                        'update:triggers',
-                        'update:users',
-                      ],
-                    },
-                  },
-                  body: {
-                    email: 'testRudderlabs+21@gmail.com',
-                    password: 'dummyPassword',
-                    connection: 'Username-Password-Authentication',
-                  },
-                  path: '/api/v2/users',
-                  query: {},
-                  method: 'post',
-                  channel: 'https://manage.auth0.com/',
-                  userAgent:
+                  user_agent:
                     'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36',
-                },
-                response: {
-                  body: {
-                    name: 'testRudderlabs+21@gmail.com',
-                    email: 'testRudderlabs+21@gmail.com',
-                    picture:
-                      'https://s.gravatar.com/avatar/0902f9d02b92aed9f0ac59aaf9475b60?s=480&r=pg&d=https%3A%2F%2Fcdn.auth0.com%2Favatars%2Fbh.png',
-                    user_id: 'auth0|dummyPassword',
-                    nickname: 'testRudderlabs+21',
-                    created_at: '2022-10-31T05:57:06.864Z',
-                    identities: [
-                      {
-                        user_id: 'auth0|dummyPassword',
-                        isSocial: false,
-                        provider: 'auth0',
+                  details: {
+                    request: {
+                      ip: '35.166.202.113',
+                      auth: {
+                        user: {
+                          name: 'rudder test',
+                          email: 'test@rudderstack.com',
+                          user_id: 'auth0|dummyPassword',
+                        },
+                        strategy: 'jwt',
+                        credentials: {
+                          jti: '571921bf7833a97efabf08d765a0ec8f',
+                          scopes: [
+                            'create:actions',
```

Comment:
scopes[] can be limited to length of 1, there's no business logic on this, test case becomes readable